Jellyfin.Api/Models/UserDtos/LrcLyricsProvider.cs

```diff
@@ -0,0 +1,117 @@
+using System;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.Dynamic;
+using System.Globalization;
+using System.Linq;
+using LrcParser.Model;
+using LrcParser.Parser;
+using MediaBrowser.Controller.Entities;
+
+namespace Jellyfin.Api.Models.UserDtos
+{
+    /// <summary>
+    /// LRC File Lyric Provider.
+    /// </summary>
+    public class LrcLyricsProvider : ILyricsProvider
```

Comment:
`LrcLyricsProvider` and `TxtLyricsProvider` should be under `MediaBrowser.Providers.Lyric`